recipes/repo.rb

```diff
@@ -551,6 +551,79 @@
   end
 end
 
+node['ros_buildfarm']['rpm_repos'].each do |dist, versions|
```

Comment:
The idea was to keep the Pulp-specific stuff separate from the general RPM stuff. If we want to just go all-in on Pulp, I can refactor this.

Reply:
Got it. I see the logic there but I don't think we have a ready alternative and I think that for now rpms == pulp is a fine expectation.

Reply:
Unfortunately the iteration doesn't line up because pulp views the three sub-repos as part of the same Pulp repo, while the directory structure doesn't reflect that. I did move it into the first level of the iteration loop in 067fb2a, though.

Reply:
At the risk of seeming mercurial do you find the first or second commit more readable? If we can't share the single compound iteration I think that readability becomes the next highest priority. I'm good either way with a slight preference toward the current approach but yield the final decision if you want to restore the first commit with the separate iteration.